joomla/joomla-cms
# [4.0] Fix tooltip for disabled grid button

Pull Request for Issue # .

### Summary of Changes

Fix tooltip for disabled grid button

### Testing Instructions
In a backend list e.g. article list. (in your single user test system)

    checkout an article as a first user = an edit article.
    leave the editing by clicking the browsers back button.
    The checked-out button shows a tool tip on hover - DO NOT CLICK IT!
    Log out as first user - log in as another user. This user must not be allowed to check in records.
    (user group setting in com.checkin -> not allowed)
    Go to the list (articles)
    Hover the checked-out button.

Quick-test: change $canCheckin to false in line 262 tmpl/articles/deafult.php
editor, $item->checked_out_time, 'articles.', $canCheckin); ?> 

![joomla-bug-is](https://user-images.githubusercontent.com/956537/66048699-4a554480-e52a-11e9-916a-59a487f46751.jpg)


### Expected result

![joomla-bug-should](https://user-images.githubusercontent.com/956537/66048740-60630500-e52a-11e9-9b59-56576b995a85.jpg)


### Actual result
see above


### Documentation Changes Required
No
